proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -24,40 +24,34 @@ However, with this proposal, by adding a new parameter, the SDL app always estab
 ### Change the app registration flow
 
 In order for the app to be always registered, the changes in SDL session establishment to app registration flow are shown below.
-Using the current library apps will not register at all without audio support. So it is assumed in this flow the app will only be using a compatible library.
+If the SDL app or HU cannot use this function, it will perform the same operation as before.
 
-![registeringApp_flow](../assets/proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress/registeringApp_flow.png)
+![registeringApp_flow](../assets/proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress/registeringApp_flow1.png)
 
 1. The developer will set their `requiresAudioSupport` to true.
 2. The library will send a `StartService` for the RPC service with a new param in the payload, `requiresAudioSupport`.
-3. Core will receive the `StartService` for the RPC service:
-    i. If `requiresAudioSupport` was set to false or not set, it will continue in flow.
-    ii. Core will check its current audio connects (BT A2DP), if it has a supported audio connect it will continue in flow.
-    iii. If there is no Audio support, Core will check if it supports the auto connect BT function. If it does it will continue in flow.
-    iv. If `requiresAudioSupport` is true and there is no supported audio methods and Core does not support the auto connect BT function, it will send a `StartServiceNAK` with the reason `No audio support available`.
-4. If it supports the auto connect BT function, a new param in the payload, `autoBTCapability` is set to true, otherwise set to false. Then if Core has continued, it will send a `StartServiceACK` with new param `autoBTCapability` set to true.
+3. Core will receive the `StartService` for the RPC service: 
+    i. Core will check it's current audio connects (BT A2DP), if it has a supported audio connect it will continue in flow.
+    ii. If there is no Audio support, core will check if it supports the auto connect BT feature. If it does it will continue in flow.
```

Comment:
```suggestion
    ii. If there is no audio support, Core will check if it supports the auto connect BT feature. If it does, it will continue in flow.
```